### PR TITLE
add libssh as build time dependency, to get sftp support

### PR DESCRIPTION
**Pull request by manschwetusCF**
sftp is still not available, which is very handy thing

### CODE REVIEW

File: kde/kdenetwork/kio-extras/kio-extras.py [hunk 1]
```diff
@@ -12,6 +12,7 @@ def setTargets(self):
     def setDependencies(self):
         self.buildDependencies["virtual/base"] = None
         self.buildDependencies["kde/frameworks/extra-cmake-modules"] = None
+        self.buildDependencies["lib/libssh"] = None
```
Comment (TheOneRing) [inline kde/kdenetwork/kio-extras/kio-extras.py]: Hi thanks for your contribution.
As we will also need the dependency during runtime it is a runtimeDependency.

**Review by TheOneRing — CHANGES_REQUESTED** [review]

File: kde/kdenetwork/kio-extras/kio-extras.py (hunk 1 above)
Comment (manschwetusCF) [reply kde/kdenetwork/kio-extras/kio-extras.py]: Changed as requested

**Comment by TheOneRing** [comment]
thx, merged

**Comment by TheOneRing** [comment]
Hi thx for your support, kio extras now detec libssh. but kio extras need some love.

```
if (NOT WIN32 AND LIBSSH_FOUND)
    # does not compile on Windows: kio_sftp.cpp(28): fatal error C1083: Cannot open include file: 'utime.h': No such file or directory
    add_subdirectory(sftp)
endif ()
```

**Pull request closed without merging**